### PR TITLE
ES6-ification and general cleanup

### PR DESCRIPTION
**Pull request by truenorth**

### CODE REVIEW

File: addon/components/md-checkbox.js [hunk 1]
```diff
@@ -13,22 +14,18 @@ var MdCheckbox = Ember.Component.extend(LayoutRules, RippleMixin, {
 
     attributeBindings: ['isDisabled:disabled', 'aria-label', 'mdNoInk', 'tabindex'],
     tabindex: 0,
+    checked: false,
+
+    didInsertElement() {
+        this._super(...arguments);
+        setupRipples(this, this.$('.md-container'));
```
Comment (mike1o1) [inline addon/components/md-checkbox.js]: This doesn't seem like the correct behavior. We want to call attachCheckboxBehavior, not attachButtonBehavior, which isn't exposed in the setupRipples utility.

File: addon/components/md-radio-button.js [hunk 2]
```diff
@@ -12,15 +13,10 @@ var MdRadioButton = Ember.Component.extend(LayoutRules, RipplesMixin, {
...
-        if (this.get('mdNoInk')) {
-            return;
-        }
-        this.get('rippleService').attachCheckboxBehavior(this.$('.md-container'));
-
-    }),
+    didInsertElement() {
+        this._super(...arguments);
+        setupRipples(this, this.$('.md-container'));
+    },
```
Comment (mike1o1) [inline addon/components/md-radio-button.js]: Same issue. This needs to attachCheckboxBehavior, not button.

File: addon/components/md-checkbox.js (hunk 1 above)
Comment (truenorth) [inline addon/components/md-checkbox.js]: Good catch. I'll export multiple symbols from the setupRipples utility.

File: addon/components/md-tabs.js [hunk 3]
```diff
@@ -207,11 +227,7 @@ var MdTabs = Ember.Component.extend(Ember.Evented, LayoutRules, RippleMixin, {
             return false;
         }
         var canvasWidth = this.$().prop('clientWidth');
-        Ember.EnumerableUtils.forEach(this.elements.tabs, function(tab) {
-            canvasWidth -= tab.offsetWidth;
-        });
-
-
+        Ember.EnumerableUtils.forEach(this.elements.tabs, tab => canvasWidth -= tab.offsetWidth);
```
Comment (mike1o1) [inline addon/components/md-tabs.js]: Would you be opposed to converting it to a full expression with curly braces? My brain has trouble parsing expressions like this when I see them, and I think having braces and a new line makes the code more readable.

**Comment by mike1o1** [comment]
Looks great aside from a minor nitpick on single line lambda methods and the ripple utility. I really like the styling of how you extracted the ember events to clean up all of the `Ember.on('didInsertElement')` type methods. Thanks!

File: addon/components/md-checkbox.js (hunk 1 above)
Comment (truenorth) [inline addon/components/md-checkbox.js]: Decided to add some functions to the existing ripples service instead. It's still substantially DRY-ed up compared to the previous implementation

File: addon/components/md-tabs.js (hunk 3 above)
Comment (truenorth) [inline addon/components/md-tabs.js]: converted to a multi-line arrow function
```js
x => {
  console.log(x);
}
```

**Comment by truenorth** [comment]
@mike1o1 I think all feedback has been addressed now. Thanks for the code review

**Pull request merged**